src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -239,6 +252,85 @@ public boolean hasParserResults() {
         return loaded;
     }
 
+    private void writeXMPtoPdf(List<ParserResult> loaded, String filesAndCitekeys, Charset encoding, XmpPreferences xmpPreferences, FilePreferences filePreferences) {
+
+        ParserResult pr = loaded.get(loaded.size() - 1);
+        BibDatabaseContext databaseContext = pr.getDatabaseContext();
+        BibDatabase dataBase = pr.getDatabase();
+
+        XmpPdfExporter xmpPdfExporter = new XmpPdfExporter(xmpPreferences);
+
+        if (filesAndCitekeys.equals("all")) {
```

Comment:
```suggestion
        if ("all".equals(filesAndCitekeys)) {
```
This prevents NPEs in case filesAndCiteKeys are somehow null

Reply:
Smart. I like it.